source/MaterialXTest/MaterialXGenShader/GenShaderUtil.h

```diff
@@ -141,16 +141,23 @@ class TestSuiteOptions
     mx::FileSearchPath externalTestPaths;
 
     // Wedge parameters
-    mx::StringVec wedgeFiles;
-    mx::StringVec wedgeParameters;
-    mx::FloatVec wedgeRangeMin;
-    mx::FloatVec wedgeRangeMax;
-    mx::IntVec wedgeSteps;
-
+    struct wedgesetting {
```

Comment:
Nitpick. use camelCase so it's "wedgeSetting, wedgeSettings, bakeSetting, bakeSettings" respectively.

Reply:
More nickpick: for a struct or class declaration it should be WedgeSetting, BakeSetting, etc. , to follow our normal coding convention.